README.md

```diff
@@ -141,6 +141,47 @@ Obviously, to test a CCD application, for example, the proper setup procedure sh
 have been completed as a prerequisite. An incomplete setup, like some users not fully 
 and correctly configured, can cause some or all of the functional tests to fail as false positives.
 
+### 3.11) Importing definition files
+
+BEFTA Framework contains definition files in both XLSX and JSON formats.
+
+Typically, CCD services under test will define implementations of a TestAutomationAdapter by extending the `befta-fw` 
+provided `DefaultTestAutomationAdaoter`, and overriding its `doLoadTestData` method in order to load this definition
+data before running feature tests.
+
+```java
+public class ServiceTestAutomationAdapter extends DefaultTestAutomationAdapter {
+
+    private final TestDataLoaderToDefinitionStore loader = new TestDataLoaderToDefinitionStore(this);
+
+    @Override
+    public void doLoadTestData() {
+        loader.addCcdRoles();
+        loader.importDefinitions();
+    }
+}
```

Comment:
This readme enhancement is good overall, but the above section will change after befta-fw 6.0. So, better not to give code-level details, perhaps. The essence of this enhanced content could perhaps be articulated without code quotations.